Formula/pandoc.rb

```diff
@@ -25,7 +25,7 @@ class Pandoc < Formula
 
   def install
     system "cabal", "v2-update"
-    system "cabal", "v2-install", *std_cabal_v2_args
+    system "cabal", "v2-install", *std_cabal_v2_args, "pandoc-cli"
```

Comment:
Note:  if you want this to behave like previous homebrew pandoc versions, you should probably add `"-fembed_data_files"`.  This causes pandoc's data files to be embedded in the binary; otherwise, they will be put on the file system somewhere.  This wasn't necessary with your previous install method because this default was specified in the `cabal.project` included in the pandoc release tarball.  But now that your install target is pandoc-cli, it will be necessary.

Reply:
@jgm - for those of us who have upgraded already, where will these file be placed (on a *nix filesystem for example)?

Reply:
I think in PREFIX/share/pandoc but it's been a while since I've installed anything that way...

Reply:
@jgm do you mind creating an improvement PR? Thanks!

Reply:
I've got enough to do on pandoc itself, I'll leave this part to you.  But my overall suggestion is:

- don't download any tarball
- use the command `cabal v2-install -fembed_data_files pandoc-cli`

Reply:
Or, even more minimally, keep what you have but add `-fembed_data_files`.

Reply:
OK, what I said above may not be correct. It looks like your current build method is baking in the data files (confirmed by installing from homebrew).  I guess that's because you're using the repository tarball which has the cabal.project file.

So, the current system should be okay, but you could replace it with what I suggest above if you don't want to download the whole repository.

Reply:
On second look, here are two further changes that would be good:

- have the installer create symlinks to the `pandoc` executable called `pandoc-lua` and `pandoc-server`, in the bin directory with `pandoc`. This will give users access to `pandoc-lua` and `pandoc-server`.
- install the man pages for `pandoc-lua.1` and `pandoc-server.1` in addition to `pandoc`. These can be found in the `man/` directory of the tarball.

Reply:
Appreciate the look, I will check it out later.